src/lib.rs

```diff
@@ -0,0 +1,36 @@
+#![crate_name = "librespot"]
```

Comment:
It's libre as in open source, not the repot library, so it's fine this way. `liblibrespot` is a bit unfortunate, but it's almost never visible, so that's alright